### ISSUE TITLE
Sample does not work

### ISSUE THREAD

**Issue by ghost** [veeso/android-media-player-standalone-library#4]
Hi, I have a problem with running android sample, it's just a black screen and nothing more, even I added link to youtube video (I registered my app in veeso and youtube, got youtube app key and set it to APSMediaPlayer instance), and I find that last version of the lib hasn't classes inside the appscendvastplayer.jar

**Comment by andreimarinescu** [comment]
Hi @ArtemUvarenko,

Is this happening with YouTube only? If so, it's because the current YouTube Android API jarfile is broken on lollipop. We've updated the jar internally and we'll push a new release soon.

If the issue is happening on usual MP4/HLS sources, please post some logcat logs / integration code and we'll have a look a.s.a.p.

**Comment by ghost** [comment]
No it's happening with other links too.
here is integration code
APSMediaBuilder builder = new APSMediaBuilder();
        try {
            builder.configureFromURL(new URL(
                    "http://clips.vorwaerts-gmbh.de/VfE_html5.mp4"));
        } catch (MalformedURLException e) {
            e.printStackTrace();
        }
        builder.debugMode = true;
        APSMediaPlayer.getInstance().init(this, false);
        APSMediaPlayer.getInstance().setSize(800, 1280);
        RelativeLayout layout = new RelativeLayout(this);
        layout.setBackgroundColor(Color.WHITE);
        layout.addView(APSMediaPlayer.getInstance().viewController());
        setContentView(layout);
        APSMediaPlayer.getInstance().playMediaUnits(builder.mediaUnits());
and I use this one version of lib https://github.com/veeso/android-media-player-standalone-library/releases/tag/1.9.23 (because higher versions have not classes inside the appscendvastpalyer.jar)
here is log
06-23 05:30:45.876: D/AndroidRuntime(1821): >>>>>> START com.android.internal.os.RuntimeInit uid 0 <<<<<<
06-23 05:30:45.877: D/AndroidRuntime(1821): CheckJNI is OFF
06-23 05:30:46.101: E/memtrack(1821): Couldn't load memtrack module (No such file or directory)
06-23 05:30:46.101: E/android.os.Debug(1821): failed to load memtrack module: -2
06-23 05:30:46.112: D/AndroidRuntime(1821): Calling main entry com.android.commands.pm.Pm
06-23 05:30:46.128: D/DefContainer(1525): Copying /data/local/tmp/veesotest.apk to base.apk
06-23 05:30:46.163: D/PackageManager(474): Renaming /data/app/vmdl1723333610.tmp to /data/app/com.example.veesotest-1
06-23 05:30:46.166: I/PackageManager(474): Running dexopt on: /data/app/com.example.veesotest-1/base.apk pkg=com.example.veesotest isa=x86 vmSafeMode=false
06-23 05:30:46.171: I/dex2oat(1835): /system/bin/dex2oat --zip-fd=5 --zip-location=/data/app/com.example.veesotest-1/base.apk --oat-fd=6 --oat-location=/data/dalvik-cache/x86/data@app@com.example.veesotest-1@base.apk@classes.dex --instruction-set=x86 --instruction-set-features=default --runtime-arg -Xms64m --runtime-arg -Xmx512m --swap-fd=7
06-23 05:30:46.208: I/dex2oat(1835): Decided to run without swap.
06-23 05:30:46.846: I/dex2oat(1835): dex2oat took 674.712ms (threads: 4) arena alloc=602KB java alloc=2MB native alloc=7MB free=8MB
06-23 05:30:46.860: V/BackupManagerService(474): restoreAtInstall pkg=com.example.veesotest token=2 restoreSet=0
06-23 05:30:46.860: V/BackupManagerService(474): Finishing install immediately
06-23 05:30:46.861: W/Settings(474): Setting install_non_market_apps has moved from android.provider.Settings.Global to android.provider.Settings.Secure, returning read-only value.
06-23 05:30:46.863: D/Documents(1729): Update found 6 roots in 1ms
06-23 05:30:46.880: I/InputReader(474): Reconfiguring input devices.  changes=0x00000010
06-23 05:30:46.883: D/BackupManagerService(474): Received broadcast Intent { act=android.intent.action.PACKAGE_ADDED dat=package:com.example.veesotest flg=0x4000010 (has extras) }
06-23 05:30:46.883: W/BackupManagerService(474): Removing schedule queue dupe of com.example.veesotest
06-23 05:30:46.899: W/VoiceInteractionManagerService(474): no available voice recognition services found for user 0
06-23 05:30:46.905: I/art(474): Explicit concurrent mark sweep GC freed 30693(2MB) AllocSpace objects, 18(1167KB) LOS objects, 33% free, 10MB/15MB, paused 763us total 42.741ms
06-23 05:30:46.906: I/art(1821): System.exit called, status: 0
06-23 05:30:46.906: I/AndroidRuntime(1821): VM exiting with result code 0.
06-23 05:30:46.916: D/Launcher.Workspace(779): 11683562 - removeExtraEmptyScreen()
06-23 05:30:46.916: D/Launcher.Workspace(779): 11683562 - convertFinalScreenToEmptyScreenIfNecessary()
06-23 05:30:46.998: D/AndroidRuntime(1848): >>>>>> START com.android.internal.os.RuntimeInit uid 0 <<<<<<
06-23 05:30:46.999: D/AndroidRuntime(1848): CheckJNI is OFF
06-23 05:30:47.022: E/memtrack(1848): Couldn't load memtrack module (No such file or directory)
06-23 05:30:47.022: E/android.os.Debug(1848): failed to load memtrack module: -2
06-23 05:30:47.039: D/AndroidRuntime(1848): Calling main entry com.android.commands.am.Am
06-23 05:30:47.044: I/ActivityManager(474): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10000000 cmp=com.example.veesotest/.MainActivity} from uid 0 on display 0
06-23 05:30:47.045: V/WindowManager(474): addAppToken: AppWindowToken{282af025 token=Token{30975c1c ActivityRecord{31f8b18f u0 com.example.veesotest/.MainActivity t17}}} to stack=1 task=17 at 0
06-23 05:30:47.046: D/AndroidRuntime(1848): Shutting down VM
06-23 05:30:47.050: V/WindowManager(474): Adding window Window{264aabd9 u0 Starting com.example.veesotest} at 3 of 8 (after Window{32b26d98 u0 com.android.launcher3/com.android.launcher3.Launcher})
06-23 05:30:47.052: I/art(1848): Debugger is no longer active
06-23 05:30:47.055: I/art(1862): Late-enabling -Xcheck:jni
06-23 05:30:47.062: I/ActivityManager(474): Start proc 1862:com.example.veesotest/u0a82 for activity com.example.veesotest/.MainActivity
06-23 05:30:50.047: D/TaskPersister(474): removeObsoleteFile: deleting file=15_task_thumbnail.png
06-23 05:30:52.224: I/art(1862): WaitForGcToComplete blocked for 5.561ms for cause HeapTrim
06-23 05:30:55.433: I/art(1862): Background sticky concurrent mark sweep GC freed 10(656B) AllocSpace objects, 8(11MB) LOS objects, 4% free, 26MB/28MB, paused 5.634ms total 9.750ms
06-23 05:30:55.492: W/art(1862): Suspending all threads took: 6.772ms
06-23 05:30:55.493: I/art(1862): Background sticky concurrent mark sweep GC freed 14(2880B) AllocSpace objects, 8(11MB) LOS objects, 21% free, 23MB/30MB, paused 7.060ms total 10.155ms
06-23 05:30:57.046: W/ActivityManager(474): Launch timeout has expired, giving up wake lock!
06-23 05:31:03.475: I/art(1862): Background sticky concurrent mark sweep GC freed 14(752B) AllocSpace objects, 8(17MB) LOS objects, 13% free, 33MB/38MB, paused 7.551ms total 15.364ms
06-23 05:31:04.071: W/art(1862): Suspending all threads took: 12.696ms
06-23 05:31:04.075: I/art(1862): Background sticky concurrent mark sweep GC freed 12(976B) AllocSpace objects, 8(16MB) LOS objects, 6% free, 36MB/39MB, paused 12.965ms total 19.661ms
06-23 05:31:16.404: W/art(1862): Suspending all threads took: 10.699ms
06-23 05:31:16.408: I/art(1862): Background sticky concurrent mark sweep GC freed 4(272B) AllocSpace objects, 4(12MB) LOS objects, 0% free, 55MB/55MB, paused 12.382ms total 18.335ms
06-23 05:31:22.243: I/art(1862): WaitForGcToComplete blocked for 7.257ms for cause HeapTrim
06-23 05:31:28.481: I/art(1862): Background partial concurrent mark sweep GC freed 23(1280B) AllocSpace objects, 12(46MB) LOS objects, 32% free, 33MB/49MB, paused 5.181ms total 15.250ms
06-23 05:31:28.498: I/art(1862): Background sticky concurrent mark sweep GC freed 4(448B) AllocSpace objects, 3(11MB) LOS objects, 0% free, 53MB/53MB, paused 5.088ms total 12.503ms
06-23 05:31:32.254: I/art(1862): WaitForGcToComplete blocked for 10.380ms for cause HeapTrim
06-23 05:31:32.276: I/art(1862): Background sticky concurrent mark sweep GC freed 5(1168B) AllocSpace objects, 3(12MB) LOS objects, 0% free, 64MB/64MB, paused 9.120ms total 15.444ms
06-23 05:31:34.704: W/art(1862): Suspending all threads took: 8.110ms
06-23 05:31:34.712: I/art(1862): Background partial concurrent mark sweep GC freed 18(1056B) AllocSpace objects, 11(44MB) LOS objects, 29% free, 37MB/53MB, paused 8.663ms total 29.697ms
06-23 05:31:34.743: I/art(1862): Background partial concurrent mark sweep GC freed 22(3KB) AllocSpace objects, 10(42MB) LOS objects, 37% free, 26MB/42MB, paused 12.906ms total 20.154ms

**Comment by andreimarinescu** [comment]
Hi @ArtemUvarenko,

I've just pushed out 2.0.5 that should solve the issues with appscendvastplayer.jar. I've tested the binary after release, let me know if it works for you as well.

Regarding the logcat post, I'm not seeing any player logs there, but the issue seems to be in the code:

configureFromUrl takes as an argument a JSON configuration file for the player (you can use configureFromData if the JSON string is already available in your code). An example of such a configuration file:

```
 {
 "content": [{
    "url": "http://clips.vorwaerts-gmbh.de/VfE_html5.mp4",
    "autoplay": true,
    "controls": {"components" : ["playback", "totalTime", "slider", "currentTime"]}
  }]
  }
```

**Comment by ghost** [comment]
thank you for replay, the problem has been solved with 2.0.5, that was my
bad I decided that I can configure builder from video link and play it later

2015-06-24 15:53 GMT+03:00 Andrei Marinescu notifications@github.com:
